### PR TITLE
JTBD Guide Review for Connect your app code to the API

### PR DESCRIPTION
**Pull request by heatheramz**
### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [ ] amplify-cli
- [ ] amplify-ui
- [ ] amplify-studio
- [ ] amplify-hosting
- [ ] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [ ] JS
- [ ] iOS
- [ ] Android
- [ ] Flutter
- [ ] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [ ] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [ ] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [ ] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [ ] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br /> 
      _ref: MDX: `[link](https://link.com)` 
            HTML: `<a href="https://link.com">link</a>`_

### When this PR is ready to merge, please check the box below
- [ ] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

File: src/pages/lib/graphqlapi/connect/q/platform/[platform].mdx [hunk 1]
```diff
@@ -2,3 +2,243 @@ export const meta = {
...
+Before you begin, you will need:
+- An GraphQL API already set up either via Amplify or via the underlying AWS AppSync service. Review [TK]() on how to build your API and database with Amplify. For guidance on how to build a GraphQL API via AWS AppSync, see [Designing a GraphQL API in AWS AppSync](https://docs.aws.amazon.com/appsync/latest/devguide/designing-a-graphql-api.html).
+- A frontend application set up. For instructions on setting up an app, see [Project Setup: Create your application](https://docs.amplify.aws/lib/project-setup/create-application/q/platform/js/)
+- [npm installed](https://docs.npmjs.com/getting-started)
+
+## Install and configure the Amplify Libraries
+
+Install the `aws-amplify` NPM package to your app. This is the main library for working with Amplify Libraries in your projects. This will include connecting your app with the GraphQL endpoint. 
+
+If you are using `Next.js`, `Vue`, or `Angular`, please review the following details for installing the Amplify Libraries:
```
Comment (heatheramz) [inline src/pages/lib/graphqlapi/connect/q/platform/[platform].mdx:21]: @renebrandel Line 37 "After reading to the bottom, I'm not sure if this makes sense to be here. I think we need to do some consolidation with the bottom blocks. Let's discuss on a call to align on the edits." 

I updated this section and moved the import config details into the block switcher below so place to start editing from. However, I did not add the yarn info b/c all switchers focus currently on npm. Also updated the order in text and switcher.

File: src/pages/lib/graphqlapi/connect/q/platform/[platform].mdx [hunk 2]
```diff
@@ -2,3 +2,243 @@ export const meta = {
...
+  bootstrap: [AppComponent]
+})
+export class AppModule { }
+```
+
+</Block>
+</BlockSwitcher>
+
+Well done! You now have your Amplify Libraries set up and configured.
+
```
Comment (heatheramz) [inline src/pages/lib/graphqlapi/connect/q/platform/[platform].mdx:139]: @renebrandel Comment "I think we should add one more step here to quickly show how customers can read/write to their database (but then let customers expand on it in the following pages). It closely follows the codegen section." 

Moved CodeGen down to review here with the added Mutate and Query data from current Getting Started Guide which is also below. This is not intended to be the final read but rather pulling all related CTAs to this area so easier to edit and review.

**Review by heatheramz — COMMENTED** [review]
@renebrandel Updated the Connect your app code to the API guide to review on its own and transferred unresolved questions from the previous PR.

File: src/pages/lib/graphqlapi/connect/q/platform/[platform].mdx [hunk 3]
```diff
@@ -2,3 +2,243 @@ export const meta = {
   title: `Connect your app code to the API`,
   description: `Learn how to connect your app code to the API.`
 };
+
```
[the remark this answers is not in the document]
Comment (renebrandel) [reply src/pages/lib/graphqlapi/connect/q/platform/[platform].mdx:5]: No dependencies for RN!

File: src/pages/lib/graphqlapi/connect/q/platform/[platform].mdx (hunk 2 above)
Comment (renebrandel) [reply src/pages/lib/graphqlapi/connect/q/platform/[platform].mdx:222]: collapse 223 - 230 into  184.

**Pull request merged**